curriculum/challenges/english/24-b1-english-for-developers/learn-how-to-describe-places-and-events/66a909dc2e382527ead44c23.md

```diff
@@ -0,0 +1,49 @@
+---
+id: 66a909dc2e382527ead44c23
+title: Task 1
+challengeType: 19
+dashedName: task-1
+---
+<!-- (Audio) Bob: Hey Lisa, do you have a minute?  -->
+
+# --description--
+
+The phrase `Do you have a minute?` is a polite way to ask if someone has time to talk.
+
+# --question--
+
+## --text--
+
+Listen to the audio and answer: What does Bob want to know from Lisa?
```

Comment:
```suggestion
# --instructions--

Listen to the audio and answer the question below.

# --question--

## --text--

What does Bob want to know from Lisa?
```

I think this should be moved the to the instructions. Or just remove it. It looks like the questions don't usually have instructions.

Reply:
I've elaborated this kind of question with a specific purpose in mind. 
My intention is to check if the student understands the purpose of Bob's question. In the options students are not selecting what they hear, but interpreting the meaning of the audio. So, In the audio Bob is saying " Hey Lisa, do you have a minute?".  If I don't ask "What does Bob want to know from Lisa?" They wouldn't know what to answer.